### PR TITLE
twa-web-docs

### PR DESCRIPTION
**Pull request by mdhillman**
This PR brings in some documentation on how to better structure URLs so that they will still resolve correctly when placed behind redirects from [theworldavatar.io](https://theworldavatar.io) website.

It also adds a simple testing ground to allow users to run a local NGINX instance with a redirect to their external service, allowing them to test their URLs before requesting a redirect from theworldavatar.io.

### CODE REVIEW

**Review by Ushcode — APPROVED** [review]

**Review by Ushcode — APPROVED** [review]

File: web/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+As an extra note, it is worth testing if your URLs require an ending slash (`/`) character to function. URLs with trailing slashes are not equivalent to those without, and depending on your service's NGINX configuration, may not return the same content.
+
+### Testing your URLs
+
+Within the `url-testing` directory, there's a small example NGINX service configured to run as a Docker container that will simulate the TWA reverse proxy so that you can access your new service container through a redirect and test these URLs.
+
+To run it, simply add the URL for your external container (which can be running) to a `.env` following the format below, then run the `run.sh` script. This should spin up a local NGINX container with a redirect to your external service, accessible via the `https://localhost/redirect/` URL.
```
Comment (gpeb2) [inline web/docs/README.md]: ```run.sh``` has been renamed.

File: web/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+> You may find that accessing the redirect in your browser shows a "Your connection is not private" error. This is expected as we're creating a fake (self-signed) SSL certificate to emulate HTTPS locally. You should be able safely dismiss and continue to test your URLs via the redirect.
+
+Expected format for `.env` file:
+
+```
+    URL=http://127.0.0.1:3000/
+```
+### Example
+
+The committed `docker-compose.yml` file also spins up a simple web server in it's own container hosting some static content to display an example of which URL types will and won't work. You can access this container directly from `http://localhost:3000` or via the provided redirect at `https://localhost/example`.
```
Comment (gpeb2) [inline web/docs/README.md]: Might be bettered as numbered steps.

File: web/docs/url-testing/start.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#
+# This script handles starting up a single (i.e. non-stack)
+# NGINX container to test redirects to an external web service.
+#
+
+# Run the system
+echo "Building and starting containers..."
+docker compose -f ./docker-compose.yml up -d --build
```
Comment (Ushcode) [inline web/docs/url-testing/start.sh:10]:
```suggestion
docker compose -f ./docker-compose.yml up -d --build || exit 1
```

Exit if the docker compose fails

**Review by Ushcode — CHANGES_REQUESTED** [review]

**Review by gpeb2 — APPROVED** [review]
I'd have probably used numbers rather than bullet points for the testing steps but it's fine as it is.

**Review by Ushcode — APPROVED** [review]

**Pull request merged**